packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js

```diff
@@ -27,7 +27,7 @@ function InputSizer({ placeholder, inputText, children }) {
 		<React.Fragment>
 			{children(width)}
 			<span style={style} ref={sizerRef}>
-				{text}
+				{text}&nbsp;
```

Comment:
Please don't use that. Instead, call `children(width + EXTRA_SPACE)` where EXTRA_SPACE is a constant defined somewhere